src/ThreeJsPanel.ts

```diff
@@ -308,6 +311,58 @@ export class ThreeJsPanel {
     this.axisCamera.position.set(-this.axisOffset[0], -this.axisOffset[1], this.axisScale * 2.0);
   }
 
+  orthoScreenPixelsToPhysicalUnits(pixels: number, physicalUnitsPerWorldUnit: number): number {
+    // At orthoScale = 0.5, the viewport is 1 world unit tall
+    const worldUnitsPerPixel = this.orthoScale * 2 / this.getHeight();
```

Comment:
Just for the sake of discussion, and fully understanding the coordinate transforms, another way to say this is that the ortho cameras are setting up a coordinate system that spans `-orthoScale` to `+orthoScale` from bottom to top (and `-orthoScale*aspect`, `+orthoScale*aspect`) from left to right.  so the total range from bottom to top = top-bottom = orthoScale - (-orthoScale) = 2.0 * orthoScale.